support/flatpak/fontsan-deps-sources.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Instead of checking these JSON files in, flatpak-updater.sh should be integrated into the flatpak build and generate these files into the `target` directory during build.